osmosis-1/osmosis.zone_assets.json

```diff
@@ -4790,8 +4790,8 @@
         {
           "name": "Penumbra Shield Funds",
           "type": "external_interface",
-          "deposit_url": "https://stake.with.starlingcyber.net/#/ibc",
-          "withdraw_url": "https://stake.with.starlingcyber.net/#/ibc"
+          "deposit_url": "https://stake.with.starlingcyber.net/#/deposit/skip",
```

Comment:
The old URL goes to 404.
I see Deposit (Manual) and Withdraw pages, but both don't seem to work well, and it seems that Deposit (Skip) is the best functioning page.
I do, however see message of a "Block sync error: ConnectError: [unknown] Failed to fetch" up at the top...